apps/meteor/server/lib/moderation/reportMessage.ts

```diff
@@ -0,0 +1,57 @@
+import { Messages, ModerationReports, Rooms, Users } from '@rocket.chat/models';
+import type { IMessage, IUser } from '@rocket.chat/core-typings';
+
+import { canAccessRoomAsync } from '../../../app/authorization/server/functions/canAccessRoom';
+import { AppEvents, Apps } from '../../../ee/server/apps';
+
+export const reportMessage = async (messageId: IMessage['_id'], description: string, uid: IUser['_id']) => {
+	if (!uid) {
+		throw new Error('error-invalid-user');
+	}
```

Comment:
do you see this happening?

Reply:
Yes, it was used in the previous method, so I kept it.

Reply:
ok then